pkg/controller/volume/attachdetach/reconciler/reconciler.go

```diff
@@ -248,6 +252,15 @@ func (rc *reconciler) reconcile() {
 			if rc.isMultiAttachForbidden(volumeToAttach.VolumeSpec) {
 				nodes := rc.actualStateOfWorld.GetNodesForVolume(volumeToAttach.VolumeName)
 				if len(nodes) > 0 {
+					if volumeToAttach.AllowMultiAttach {
+						simpleMsg, detailedMsg := volumeToAttach.GenerateMsg("Multi-Attach error", "Volume is already exclusively attached to one node and can't be attached to another")
+						for _, pod := range volumeToAttach.ScheduledPods {
+							rc.recorder.Eventf(pod, v1.EventTypeWarning, "MultiAttachForbidden", simpleMsg)
```

Comment:
nit: How about just `FailedAttach` instead of `MultiAttachForbidden` and stick it in a const.